terraform/modules/monitoring/monitoring.tf

```diff
@@ -0,0 +1,181 @@
+variable "gcp_project" {
+  type = string
+}
+
+variable "gcp_region" {
+  type = string
+}
+
+variable "environment" {
+  type = string
+}
+
+resource "kubernetes_namespace" "monitor" {
+  metadata {
+    name = "monitor"
+  }
+}
+
+resource "kubernetes_service" "prometheus" {
+  metadata {
+    name      = "prometheus"
+    namespace = "monitor"
+  }
+  spec {
+    selector = {
+      app = kubernetes_pod.prometheus.metadata.0.labels.app
+    }
+    port {
+      port        = 9090
+      target_port = 9090
```

Comment:
Could you add a comment explaining what is exposed on these ports? Is this the Prometheus API?